package.json

```diff
@@ -0,0 +1,91 @@
+{
+  "name": "esplora-api-client",
+  "version": "1.0.0",
+  "description": "Tiny client library for Esplora-based Bitcoin APIs",
+  "keywords": [
+    "api",
+    "bitcoin",
+    "blockstream",
+    "client",
+    "esplora",
+    "mempool",
+    "mempool.space"
+  ],
+  "homepage": "https://github.com/hemilabs/esplora-api-client",
+  "bugs": {
+    "url": "https://github.com/hemilabs/esplora-api-client/issues"
+  },
+  "license": "MIT",
+  "author": "Gabriel Montes <gabriel@bloq.com> (https://github.com/gabmontes)",
+  "files": [
+    "src/**/*.js"
+  ],
+  "main": "src/index.js",
+  "repository": "hemilabs/esplora-api-client",
+  "scripts": {
+    "coverage": "c8 npm test",
+    "format:check": "prettier --check .",
+    "knip": "knip",
+    "lint": "eslint --cache .",
+    "prepare": "husky",
+    "prepublishOnly": "npm run build",
+    "test": "mocha"
+  },
+  "dependencies": {
+    "fetch-plus-plus": "^1.0.0"
+  },
+  "devDependencies": {
+    "@types/node": "^22.7.3",
+    "c8": "^10.1.2",
+    "chai": "^5.1.1",
+    "chai-as-promised": "^8.0.0",
+    "eslint": "^8.57.1",
+    "eslint-config-bloq": "^4.1.0",
+    "husky": "^9.1.6",
+    "knip": "^5.30.5",
+    "lint-staged": "^15.2.10",
+    "mocha": "^10.7.3",
+    "nock": "^14.0.0-beta.14",
+    "prettier": "^3.3.3",
+    "typescript": "^5.6.2"
+  },
+  "type": "module",
+  "c8": {
+    "all": true
+  },
+  "eslintConfig": {
+    "extends": [
+      "bloq",
+      "prettier"
+    ],
+    "overrides": [
+      {
+        "files": [
+          "*.test.js"
+        ],
+        "extends": [
+          "bloq/mocha",
+          "prettier"
+        ],
+        "rules": {
+          "node/no-unpublished-require": "off"
+        }
+      }
+    ],
+    "parserOptions": {
+      "sourceType": "module"
+    },
+    "root": true
+  },
+  "knip": {
+    "ignoreDependencies": [
+      "eslint-config-prettier",
+      "lint-staged"
+    ]
+  },
+  "lint-staged": {
+    "*.{js,md}": "eslint --cache --fix",
+    "*.{js,json,md}": "prettier --write"
+  },
+  "prettier": {}
```

Comment:
is this needed for the project to pick up prettier's defaults? Or can it be removed?

Reply:
Needed so VSCode users can keep the Prettier plugin "Require Config" set and auto-format the code.